client/my-sites/preview/main.jsx

```diff
@@ -27,10 +30,28 @@ class PreviewMain extends React.Component {
 
 	state = {
 		previewUrl: null,
+		showingClose: false,
 	};
 
 	componentWillMount() {
 		this.updateUrl();
+		this.updateLayout();
+	}
+
+	updateLayout = () => {
+		this.setState( {
+			showingClose: isWithinBreakpoint( '<660px' ),
+		} );
+	}
+
+	throttledUpdateLayout = throttle( this.updateLayout, 100 );
```

Comment:
Could a trailing `debounce` be better here? that way we wouldn't bother attempting an update during resizing (in cases where resizing takes longer than 100ms).
I think the only people who spend longer than that resizing are actually devs or designers so maybe not!